src/db.js

```diff
@@ -0,0 +1,13 @@
+const { Sequelize } = require('sequelize');
+
+const client = new Sequelize({
```

Comment:
The Sequelize instance is created without specifying the `port` option. If your PostgreSQL server is running on a non-default port, you should include the `port` configuration in the Sequelize instance.